contrib/automation_tests/test_cases/symbol_locations.py

```diff
@@ -92,3 +92,19 @@ def _execute(self):
         self.expect_eq(0, len(symbol_path_list.descendants(control_type='ListItem')),
                        'List is empty')
         self.find_control('Button', 'Done', parent=ui).click_input()
+
+class ToggleEnableStadiaSymbolStore(E2ETestCase):
+    """
+    Set the "Enable Stadia symbol store" option. 
+
+    If the symbol locations window is not open, it will be opened automatically. Regardless of the previous
+    state, that window will be closed after the test is done.
+    """
+
+    def _execute(self, enable_stadia_symbol_store: bool = True):
+        ui = _show_and_get_symbol_location_ui(self.suite.top_window())
+        checkbox = self.find_control('CheckBox', 'EnableStadiaSymbolStoreCheckBox', parent=ui)
+        if checkbox.get_toggle_state() != enable_stadia_symbol_store:
+            logging.info('Toggling "Enable Stadia symbol store" checkbox to {}.'.format(enable_stadia_symbol_store))
+            checkbox.click_input()
+        self.find_control('Button', 'Done', parent=ui).click_input()
```

Comment:
I just commneted the same thing on another PR:

This is probaby missing some sort of wait so that we can guarantee the dialog is closed before we exit this method.
The dialog might otherwise swallow the alt+f4 to end Orbit.

Let's chat with Pablo to see if we find a more general solution for that - this seems to show up again and again.

Reply:
Thanks for the suggestion, I added a wait for close at the end.

Reply:
As discussed offline - LGTM!